pkg/keycloak/keycloak.go

```diff
@@ -197,7 +191,10 @@ func (h *Handler) Handle(ctx context.Context, event sdk.Event) error {
 		}
 
 	case v1alpha1.PhaseComplete:
-		return h.reconcileResources(kcCopy)
+		err := h.reconcileResources(kcCopy)
+		if err != nil {
+			return errors.Wrap(err, "could not reconcile resources")
+		}
```

Comment:
Don't return here since reconcileResources might update the kcCopy object and we can leave it up to the normal update routine to call sdk update if it's modified.